source/appModules/windowsinternal_composableshell_experiences_textinput_inputapp.py

```diff
@@ -173,10 +173,12 @@ def event_UIA_window_windowOpen(self, obj, nextHandler):
 			return
 		# #9104: different aspects of modern input panel are represented by automation iD's.
 		childAutomationID = obj.firstChild.UIAElement.cachedAutomationID
-		# Emoji panel for build 16299 and 17134.
+		# Emoji panel for 1709 (build 16299) and 1803 (17134).
+		emojiPanelInitial = winVersion.WIN10_1709
 		# This event is properly raised in build 17134.
+		emojiPanelWindowOpenEvent = winVersion.WIN10_1803
 		if (
-			winVersion.getWinVer().build <= 17134
+			winVersion.getWinVer() in (emojiPanelInitial, emojiPanelWindowOpenEvent)

```

Comment:
I may be misunderstanding (I'm not that familiar with the timeline of the emoji panel), but wouldn't this be more accurate as a comparison for a range of values eg `emojiPanelInitial  <= winVersion.getWinVer() <= emojiPanelWindowOpenEvent`. Or is it actually accurate that only for these two precise builds this alternative behavior was required.

I think generally it would be safer to do these comparisons as ranges with limits where MS made / removed things.

I ask because there are other "release" builds between, but also in case there could have been (or even could still be) people running insider builds that may still benefit from this that don't exactly equal any of the reference build numbers listed in the constants. For instance running 17763: https://blogs.windows.com/windows-insider/2018/09/18/announcing-windows-10-insider-preview-build-17763/